### PR TITLE
Update message_module.cc

### PR DESCRIPTION
**Pull request by ThomasColthurst**
Add missing piece from pull request #4698:  actually initialize the new API object.

### CODE REVIEW

**Comment by ThomasColthurst** [comment]
Ping.

File: python/google/protobuf/pyext/message_module.cc [hunk 1]
```diff
@@ -130,6 +130,15 @@ extern "C" {
       Py_DECREF(m);
       return INITFUNC_ERRORVAL;
     }
+    
+    // Adds the C++ API
+    if (PyObject* api =
+            PyCapsule_New(new ApiImplementation(),
+                          proto2::python::PyProtoAPICapsuleName(), NULL)) {
```
Comment (anandolee) [inline python/google/protobuf/pyext/message_module.cc]: namespace is not correct

**Comment by ThomasColthurst** [comment]
Done.

**Comment by anandolee** [comment]
The kokoro failures are not related to the PR. Will merge

**Pull request merged**